### PR TITLE
DTD-294 - Add new pega endpoints to stub

### PR DESCRIPTION
**Pull request by TevynAllen**

### CODE REVIEW

**Review by rogerball — APPROVED** [review]

File: conf/app.routes [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # microservice specific routes
 
-GET        /hello-world                                                 uk.gov.hmrc.debttransformationstub.controllers.MicroserviceHelloWorldController.hello
 GET        /debt                                                        uk.gov.hmrc.debttransformationstub.controllers.CaseManagementSystemController.getList
 GET        /debt/:debtID                                                uk.gov.hmrc.debttransformationstub.controllers.CaseManagementSystemController.getCaseDetails(debtID: String, duties: Option[String])
```
Comment (thadeusssewanyana) [inline conf/app.routes:4]: debtID change to debtId in the sol service?

File: conf/resources/data/debt/debt001.json [hunk 2]
```diff
@@ -1,38 +1,50 @@
...
-    },
-    {
-      "debtRespiteFrom": "2021-04-04",
-      "debtRespiteTo": "2021-06-04"
-    }
-  ]
+{
+  "debtItems": [
+    {
+      "dutyID": "duty01",
```
Comment (thadeusssewanyana) [inline conf/resources/data/debt/debt001.json:4]: are we changing this to dutyId?

Comment (TevynAllen) [reply conf/resources/data/debt/debt001.json:4]: this is based on the responses given to us by pega

File: conf/app.routes (hunk 1 above)
Comment (TevynAllen) [reply conf/app.routes:4]: Tbh this endpoint won't be used, I will remove once we pass testing

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**